### ISSUE TITLE
Function th is not working:

### ISSUE THREAD

**Issue by JanVomlel** [Kotlin/kotlinx.html#41]
Function th is not working, it generates ClassCastException, version 0.6.4.

```kotlin
import kotlinx.html.js.*
import kotlinx.html.dom.create
import kotlin.browser.document

fun main(args: Array<String>) {
    val td = document.create.td()
    val th = document.create.th()
    println("OK")
}
```

**Comment by cy6erGn0m** [comment]
the reason is that is was accidentally assigned to `HTMLTableColElement` instead of `HTMLTableHeaderCellElement`

**Comment by cy6erGn0m** [comment]
Fixed in 0.6.6
